packages/wrangler/src/index.tsx

```diff
@@ -435,6 +435,20 @@ export async function main(argv: string[]): Promise<void> {
           type: "string",
           // TODO: get choices for the toml file?
         })
+        .option("compatibility-date", {
+          describe: "Date to use for compatibility checks",
+          type: "string",
+        })
+        .option("compatibility-flags", {
+          describe: "Flags to use for compatibility checks",
+          type: "array",
+          alias: "compatibility-flag",
+        })
+        .option("latest", {
```

Comment:
Should we error if `compatibility-date` and `latest` are both used?
Should we log a warning if there is a compatibility-date set in config and is being overridden by `--latest`?

Reply:
I... hrmm. I'd push back against the error; the use case is the command's been generated by some otehr tooling, or literally just inside an npm script, and you're overriding it manually to try the latest runtime version. Would be annoying if an error blocked you then. We have the warning anyway,  and it's an annoying one that you can't miss.